### DISCUSSION TITLE
A GDAL API version must be specified.

### DISCUSSION THREAD

**Discussion by MiriamSch** [corteva/rioxarray#483] [Q&A] [answered]
I am trying to install rioxarray using pip with the following command in my virtual environment: "pip install rioxarray". However, I get the following error message and do not know how to fix it: "ERROR: A GDAL API version must be specified. Provide a path to gdal-config using a GDAL_CONFIG environment variable or use a GDAL_VERSION environment variable." It would be great if someone could help me!

**Comment by snowman2** [comment, marked as answer]
That sounds like an issue with installing rasterio (https://github.com/rasterio/rasterio/). Can you provide this information?
 - Python version (`python -c "import sys; print(sys.version.replace('\n', ' '))"`)
 - Operation System Information (`python -c "import platform; print(platform.platform())"`)

Also, do you have GDAL installed on your system? If so, do you know where?

**Reply by MiriamSch** [reply]
Version: 3.8.10 (tags/v3.8.10:3d8993a, May  3 2021, 11:48:03) [MSC v.1928 64 bit (AMD64)]
Operation System: Windows-10-10.0.19043-SP0
Yes, I have GDAL installed on my system (when checking python pip list). The name of the folder should be C:\Users\Miriam\.pyvenv\ve1\Lib\site-packages\osgeo.

**Reply by snowman2** [reply]
Here are the instructions for windows rasterio installation:
https://github.com/rasterio/rasterio/#windows-1

You could also try the wheels from here:
https://www.lfd.uci.edu/~gohlke/pythonlibs/#rasterio

**Reply by MiriamSch** [reply]
it worked with the wheels from gohlke! thank you so much for your help.
